### PR TITLE
Improve files selector

### PR DESCRIPTION
**Pull request by ferazog**
Select file it's available.
Filename is showing in the component.

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt [hunk 1]
```diff
@@ -222,8 +236,32 @@ class FileSelectorField @JvmOverloads constructor(
...
                 is FileSelectorValue.DrawableValue -> view.setImageDrawable(fileSelectorValue.drawable)
                 is FileSelectorValue.BitmapValue -> view.setImageBitmap(fileSelectorValue.bitmap)
+                is FileSelectorValue.FileValue -> setFileIconInView(view, fileSelectorValue)
+            }
+        }
+    }
+
+    private fun setFileIconInView(view: AppCompatImageView, fileValue: FileSelectorValue.FileValue) {
+        val extension = with(fileValue.filepath) {
+            val index = lastIndexOf(".") + 1
```
Comment (leidyzulu) [inline app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: what is +1?

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt [hunk 2]
```diff
@@ -222,8 +236,32 @@ class FileSelectorField @JvmOverloads constructor(
...
+            EXTENSION_PDF -> view.setImageResource(R.drawable.ic_file_pdf)
+            EXTENSION_DOC, EXTENSION_DOCX -> view.setImageResource(R.drawable.ic_file_doc)
+            EXTENSION_JPEG, EXTENSION_JPG, EXTENSION_PNG -> {
+                val bitmap = BitmapFactory.decodeFile(fileValue.filepath)
+                view.setImageBitmap(bitmap)
+                view.requestLayout()
             }
+            else -> view.setImageResource(R.drawable.ic_file_base)
         }
+        tvFilename?.text = fileValue.filename ?: fileValue.filepath
```
Comment (leidyzulu) [inline app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: ya tu sabes

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt (hunk 1 above)
Comment (ferazog) [reply app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: It's `FILE_AFTER_DOT_INDEX`.

File: app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt (hunk 2 above)
Comment (ferazog) [reply app/src/main/java/co/condorlabs/customcomponents/fileselectorview/FileSelectorField.kt]: Yeah.

File: app/src/main/res/layout/file_selector_view.xml [hunk 3]
```diff
@@ -1,71 +1,82 @@
...
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/clContent" />
+            android:id="@+id/tvError"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/padding_small"
+            android:layout_marginTop="@dimen/padding_small"
+            android:layout_marginEnd="@dimen/padding_small"
+            android:text="@string/file_selector_default_error"
+            android:textColor="#ff0000"
```
[the remark this answers is not in the document]
Comment (ferazog) [reply app/src/main/res/layout/file_selector_view.xml]: Done.

File: test/src/main/res/layout/activity_file_selector_file_test.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <co.condorlabs.customcomponents.fileselectorview.FileSelectorField
+            android:id="@+id/fileSelectorOptionFile"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:tap_button_title="This is the Title"
+            app:dialog_title="Options"
```
[the remark this answers is not in the document]
Comment (ferazog) [reply test/src/main/res/layout/activity_file_selector_file_test.xml]: This is an string for testing.

**Pull request merged**